demos/bdf-suite.py

```diff
@@ -4,29 +4,24 @@
 
 """
 
+import os
+import time
+
+import pandas as pd
+
 import palimpzest as pz
 from palimpzest.utils import udfs
-import pandas as pd
-import time
-import os
 
 pz.DataDirectory().clearCache(keep_registry=True)
 
 
+
+import matplotlib.pyplot as plt
 import networkx as nx
 import streamlit as st
-import context
```

Comment:
See above

Reply:
Should this be reverted now or will the context handling happen in a follow up?

Reply:
In addition to import fixes for demos, the demos rely on packages not present in the pip install (this is more obvious when using a venv for the installation rather than global python which users of might have installed). I think it is worth giving the demos an install and run script that create a .venv folder with the requirements for all demos installed into it and then run the relevant demo via something like `my_script --demo_name` (would make demos more easily reproducible).

That said, that should be a follow up if at all desired

Reply:
I'm adding a function to do the environment loading, will push in a minute